CMakeLists.txt

```diff
@@ -38,7 +38,7 @@ if("${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_BINARY_DIR}")
     message(FATAL_ERROR "Aktualizr does not support building in the source tree. Please remove CMakeCache.txt and the CMakeFiles/ directory, then create a subdirectory to build in: mkdir build; cd build; cmake ..")
 endif()
 
-set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake-modules)
+set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake-modules;${PROJECT_SOURCE_DIR}/aktualizr/cmake-modules")
```

Comment:
Should this extra bit get set in aktualizr-android? `${PROJECT_SOURCE_DIR}/aktualizr` doesn't exist in this repo, so it seems quite odd.

Reply:
Yes, you're right. This shouldn't be there.